fvtest/compilerunittest/p/BinaryEncoder.cpp

```diff
@@ -4355,6 +4355,10 @@ INSTANTIATE_TEST_CASE_P(StoreIndex, PPCMemSrc1EncodingTest, ::testing::ValuesIn(
     std::make_tuple(TR::InstOpCode::stwbrx,  MemoryReference(TR::RealRegister::gr31, TR::RealRegister::gr0 ), TR::RealRegister::gr0,   0x7c1f052cu, false),
     std::make_tuple(TR::InstOpCode::stwbrx,  MemoryReference(TR::RealRegister::gr0,  TR::RealRegister::gr31), TR::RealRegister::gr0,   0x7c00fd2cu, false),
     std::make_tuple(TR::InstOpCode::stwbrx,  MemoryReference(TR::RealRegister::gr0,  TR::RealRegister::gr0 ), TR::RealRegister::gr31,  0x7fe0052cu, false),
+    std::make_tuple(TR::InstOpCode::stdbrx,  MemoryReference(TR::RealRegister::gr0,  TR::RealRegister::gr0 ), TR::RealRegister::gr0,   0x7c00050cu, false),
+    std::make_tuple(TR::InstOpCode::stdbrx,  MemoryReference(TR::RealRegister::gr31, TR::RealRegister::gr0 ), TR::RealRegister::gr0,   0x7c1f050cu, false),
+    std::make_tuple(TR::InstOpCode::stdbrx,  MemoryReference(TR::RealRegister::gr0,  TR::RealRegister::gr31), TR::RealRegister::gr0,   0x7c00fd0cu, false),
+    std::make_tuple(TR::InstOpCode::stdbrx,  MemoryReference(TR::RealRegister::gr0,  TR::RealRegister::gr0 ), TR::RealRegister::gr31,  0x7fe0050cu, false),
```

Comment:
The expected results for these tests look incorrect to me... According to the ISA document and output from gcc, it seems to me that `stdbrx 0, 0, 0` should be encoding as `0x7c000528u`.

Reply:
Fixed in 6fddaab. I've disassembled online via [1].

Input:
```
7c 00 05 28
7c 1f 05 28
7c 00 fd 28
7f e0 05 28
```

Output:
```
.data:00000000 7c000528        stdbrx r0,0,r0
.data:00000004 7c1f0528        stdbrx r0,r31,r0
.data:00000008 7c00fd28        stdbrx r0,0,r31
.data:0000000c 7fe00528        stdbrx r31,0,r0
```

[1] https://onlinedisassembler.com/odaweb/